### PR TITLE
Updatet SetPlayerHealthRechargeLimit

### PR DESCRIPTION
**Pull request by Korgron**
Added further information to provide more understanding of how to use this native properly

### CODE REVIEW

File: PLAYER/SetPlayerHealthRechargeLimit.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ aliases: ["0xC388A0F065F5BC34"]
...
 ```
 
-
 ## Parameters
-* **player**: 
-* **limit**: 
+- **player**: PlayerId()
+- **limit**: float number between 0.1 and 0.9
+
+## Important
```
Comment (technetium-cfx) [inline PLAYER/SetPlayerHealthRechargeLimit.md:16]: Documentation does not support additional headings. Please include this as description.

**Review by technetium-cfx — CHANGES_REQUESTED** [review]
Thanks for the contribution! Do have any code for us to verify this change?

**Comment by Korgron** [comment]
## Example code as client script in Javascript
```JS
Wait = (ms) => new Promise(resolve => setTimeout(resolve, ms))

RegisterCommand("setHealth", async () => {
    let ped = PlayerPedId()
    SetEntityMaxHealth(ped, 500) //Set Health over the default health amount to test, if the health will be recharged to 90% (0.9)
    SetPlayerHealthRechargeMultiplier(PlayerId(), 20.0) //Speed up recharging health
    SetPlayerHealthRechargeLimit(PlayerId(), 0.9) //Set rechargelimit to 0.9 (90%)
})
//Our health should be recharged to 450HP

setTick(async () => {       //create an async loop. Similar to Citizen.CreateThread(function() end)
    let ped = PlayerPedId()
    let health = GetEntityHealth(ped)
    console.log(health)  //log the amount of health to the console
    await Wait(1000)    // Wait every one second
})
```

**Comment by Disquse** [comment]
Why do you use `setTick` with 1000 delay when you can just create an interval?

**Comment by Korgron** [comment]
Well, I haven't used `setInterval()` yet, but I will give this a try

**Review by Korgron — COMMENTED** [review]
I have removed the ##Important headline and added "Description of the native"

**Comment by technetium-cfx** [comment]
Huh, that change makes no sense. Why'd you add a bare "Description of the native"? The point of the example in the contribution guidelines is to highlight where the description goes, lol 😛

**Comment by Korgron** [comment]
Oh, lol, okay, then I just remove it 😄

**Pull request closed without merging**